php-practice_1.php

```diff
@@ -1,43 +1,143 @@
 <?php
 
+// Q1
+
+$name='畠山';
+echo '私の名前は「'.$name.'」です';
 
 // Q2 四則演算
+$num = 5 * 4;
 
+echo $num;
+echo "\n";
+echo $num / 2;
 
 // Q3 日付操作
+$today = date("Y年m月d日 H時i分s秒");
 
-
+echo '現在時刻は、' . $today . 'です。';
 
 // Q4 条件分岐-1 if文
+$device = 'w';
 
+if ($device === 'windows'|| $device === 'mac'){
+    echo'使用OSは、' . $device . 'です。';
+}else {
+  echo'どちらでもありません。';
+      }
 
 // Q5 条件分岐-2 三項演算子
+$age = 5;
+
+$message = ($age > 18) ? '成人です' : '未成年です';
+
+echo $message;
 
 // Q6 配列
+$prefecture = ['群馬','茨城','栃木','千葉','埼玉','東京','神奈川','前橋市','水戸市','宇都宮市','さいたま市','千葉市','横浜市','新宿区'];
 
+echo " $prefecture[2]と$prefecture[3]は関東地方の都道府県です。";
 
 // Q7 連想配列-1
-
+$prefecture = [
+  '東京' =>'新宿区',
+  '神奈川' =>'横浜市',
+  '千葉' =>'千葉市',
+  '埼玉' =>'さいたま市',
+  '栃木' =>'宇都宮市',
+  '群馬' =>'前橋市',
+  '茨城' =>'水戸市',
+];
+
+foreach ($prefecture as $key =>$value)
+{
+      echo $value . "\n";
+}
 
 // Q8 連想配列-2
-
+foreach ($prefecture as $key =>$value)
+{
+  if($key ==='埼玉' ){
+    echo '埼玉県の県庁所在地は、さいたま市です';
+      }
+}
```

Comment:
[nits]
インデントずれているので修正お願いします！
```suggestion
foreach ($prefecture as $key => $value)
{
     if ($key === '埼玉' ){
        echo '埼玉県の県庁所在地は、さいたま市です';
      }
}
```